examples/bots/discord-bot/src/main.rs

```
@@ -0,0 +1,170 @@
+    #[autocomplete = "poise::builtins::autocomplete_command"]
+    command: Option<String>,
+) -> anyhow::Result<()> {
+    poise::builtins::help(
+        ctx,
+        command.as_deref(),
+        poise::builtins::HelpConfiguration::default(),
+    )
+    .await?;
+    Ok(())
+}
+
+/// Verify with Concordium
+#[poise::command(slash_command, prefix_command)]
+async fn verify(ctx: Context<'_>) -> anyhow::Result<()> {
+    ctx.send(|reply| {
+        reply
+            .content("Please verify with your wallet.")
+            .components(link_button(&ctx.data().verifier_url, "Verify"))
+            // Only the recipient can see the message
+            .ephemeral(true)
+    })
+    .await?;
+    Ok(())
+}
+
+/// Checks the verification status of a user
+#[poise::command(slash_command, prefix_command)]
+async fn check(
+    ctx: Context<'_>,
+    #[description = "Selected user"] user: serenity::User,
+) -> anyhow::Result<()> {
+    match get_verifications(ctx.data(), user.id).await {
+        Ok(verifications) => {
+            let mention = user.mention();
+            let message = if verifications.is_empty() {
+                format!("{mention} is not verified with Concordium.")
+            } else {
+                let mut message = format!("{mention} is verified with Concordium.");
+                for verification in verifications
+                    .into_iter()
+                    .filter(|v| v.platform != Platform::Discord && !v.revoked)
+                {
+                    message.push_str(&format!(
+                        "\n- {}: {}",
+                        verification.platform, verification.username
+                    ));
+                }
+                message
+            };
+            // .ephemeral(true) means only the recipient can see the message
+            ctx.send(|reply| reply.content(message).ephemeral(true))
+                .await?;
+        }
+        Err(err) => tracing::error!("{err}"),
+    }
+
+    Ok(())
```

What is the meaning of this OK response? It seems we ignore the error if we cannot get verifications, which is an odd choice.